### DISCUSSION TITLE
button colors when type is other than solid

### DISCUSSION THREAD

**Discussion by Pkcarreno** [react-native-elements/react-native-elements#3681] [General]
when I declare a button with a color different from the primary color and set it to a type other than solid, the button is displayed as it corresponds to the type but is displayed in the primary color no matter what color I set

Example:

``` tsx
<Button
  title="Test Button"
  color="warning"
  type="outline"
  />
```

**Comment by ibizabroker** [comment]
Maybe change the border and text color separately.
```
<Button
  title="Test Button"
  type="outline"
  buttonStyle={{
    borderColor: '#000'
  }}
  titleStyle={{ 
    color: '#000'
  }}
/>
```
